docs/blog/2016-9-28.md

```diff
@@ -0,0 +1,15 @@
+# New Release
+We continue our montly release cycle with our [September release](https://github.com/vmware/docker-volume-vsphere/releases/tag/0.7)
+
+Thanks to [Bruno Moura's](https://github.com/brunotm) excellent [work](https://github.com/vmware/docker-volume-vsphere/pull/597), Docker volume plugin for vSphere now supports selection of file system format when creating a new docker volume.
```

Comment:
nit: drop "excellent" please - the work is indeed excellent but IMO not needed for the announcement